static/src/javascripts/projects/common/modules/experiments/ab.js

```diff
@@ -117,7 +117,7 @@ define([
             isSensitive = config.page.shouldHideAdverts;
 
         return ((isSensitive ? test.showForSensitive : true)
-        && test.canRun() && !expired && isTestSwitchedOn(test));
```

Comment:
if the `canRun` throws an exception, we need to be able to turn off the test to suppress the error.